src/site/content/en/fast/minify-css/index.md

```diff
@@ -0,0 +1,184 @@
+
+As a first step, you need to understand what would be the opportunity after minifying this file:
+
+1. Open [Lighthouse](https://web.dev/measure). 
+1. Enter the URL: https://fav-kitties-animated.glitch.me/ and click **Run Audit**. 
+1. Click **View report**.
+1. Click on **Performance** and go the **Opportunities** section.
+
+The resulting report shows that up to **16KB** can be saved from the **animate.css** file:
+
+<img class="screenshot" width="700px" height="150px" src="./lighthouse-unoptimized.png" alt="Lighthouse - Minify CSS opportunity.">
+
+Now you'll inspect the content of the CSS:
+
+1. Open the [site](https://fav-kitties-animated.glitch.me/) in Chrome (it might take a while for Glitch servers to respond the first time).
+1. Press `Control+Shift+J` or `Cmd+Option+J` (Mac), to open DevTools.
+1. Click on the **Network** panel and filter for **CSS**.
+1. Make sure **Disable Cache** is checked and reload the page.
+
+<img class="screenshot" width="700px" height="120px" src="./cdt-css-unoptimized.png" alt="DevTools CSS unoptimized trace">
+
+The page is requesting two CSS files, of **1.9KB** and **76.2KB** respectively. 
+
+1. Click **animate.css**.
+1. Click the **Response** tab, to see the file contents. 
+
+Note that the stylesheet contains characters for whitespaces and indentation:
+
+<img class="screenshot" width="700px" height="120px" src="./cdt-css-unoptimized-res.png" alt="DevTools CSS unoptimized response">
+
+Next, you'll add some Webpack plugins to your build process to minify these files.
+
+<div class="aside note">
+<strong>Note:</strong> The previous Lighthouse report only lists <strong>animate.css</strong> as an opportunity for minification. Minifying <strong>style.css</strong> will also save some bytes, but not enough for Lighthouse to consider it a significant savings. However, minifying CSS is a general best practice; so it makes sense to minify all of your CSS files.
+</div>
+
+## CSS Minification with Webpack:
+
+Before jumping into the optimizations, take some time understanding how build process for the [demo site](https://glitch.com/edit/#!/fav-kitties-animated?path=webpack.config.js:1:0]) works:
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    allow="geolocation; microphone; camera; midi; vr; encrypted-media"
+    src="https://glitch.com/embed/#!/embed/fav-kitties-animated?path=webpack.config.js&previewSize=0"
+    alt="fav-kitties-animated on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+By default, the resulting JS bundle that Webpack produces would contain the content of the CSS files inlined. Since we want to maintain separate CSS files, we are using two complementary plugins:
```

Comment:
```suggestion
By default, the resulting JS bundle that webpack produces would contain the content of the CSS files inlined. Since we want to maintain separate CSS files, we are using two complementary plugins:
```